packages/webapp/src/components/LocationDetailLayout/PointDetails/index.js

```diff
@@ -57,14 +58,15 @@ export default function PointDetailsLayout({
       />
 
       {children}
-      <Input
-        label={t('common:NOTES')}
-        type="text"
-        optional
-        style={{ marginBottom: '40px' }}
-        hookFormRegister={register(pointEnum.notes)}
-        disabled={isViewLocationPage}
-      />
+      <InputAutoSize optional={true} label={t('common:NOTES')} style={{ marginBottom: '40px'}} hookFormRegister={register(pointEnum.notes)} disabled={isViewLocationPage}/>
+      {/*<Input*/}
+      {/*  label={t('common:NOTES')}*/}
+      {/*  type="text"*/}
+      {/*  optional*/}
+      {/*  style={{ marginBottom: '40px' }}*/}
+      {/*  hookFormRegister={register(pointEnum.notes)}*/}
+      {/*  disabled={isViewLocationPage}*/}
+      {/*/>*/}
```

Comment:
Remove commented code, and `InputAutoSize` space props across multiple lines

Reply:
Should do this for `LocationDetailLayout/AreaDetails/index.js` and `LocationDetailLayout/LineDetails/index.js` as well